study.md

```diff
@@ -46,11 +50,15 @@ Experiment in pry and place your answer below.
 4. Even better if you can do so while removing the two `- -`'s between `general` and `assembly`
 
 ```md
-<!-- your answer here -->
+str.reverse!
+str.split(%r{\s*})
+str.each_char {|c| print c,"""-"}
```

Comment:
This iteration block prints each character of string with a hyphen between the characters but it does not modify the original variable.